tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -375,53 +391,43 @@
         flex-direction: row-reverse;
       }
 
-      .tf-category-pane {
-        background-color: white;
-        border: 1px solid lightgrey;
-        cursor: pointer;
-        width: 100%;
-        font-size: 15px;
-        line-height: 1;
-        box-shadow: 0 1px 5px rgba(0,0,0,0.2);
-        padding: 10px 15px;
-        text-align: left;
-      }
-
       .tf-category-pane-content {
-        padding: 15px;
-        border: 1px solid #dedede;
+        flex-wrap: wrap;
+        flex-grow: 1;
+        padding: 12px;
+        border: 1px solid #dadce0;
         border-top: none;
         border-bottom-left-radius: 2px;
         border-bottom-right-radius: 2px;
         background: white;
       }
 
       :host vz-line-chart {
-        height: 120px;
-        width: 200px;
+        margin: 12px 12px 12px 24px;
```

Comment:
how does this affect roc curves?

Reply:
Adds consistent spacing around vz-line-chart for positioning.